sentry_sdk/integrations/logging.py

```diff
@@ -181,11 +183,11 @@ def _emit(self, record):
         if not self._can_record(record):
             return
 
-        hub = Hub.current
-        if hub.client is None:
+        client = sentry_sdk.get_client()
+        if not client.is_active():
             return
```

Comment:
Not 100% sure but it might be we don't need this check since if the current client is a noop client, you can still do the stuff we're doing with it, it'll just all be noop.

Reply:
I am not sure, there is a lot of code after this line that we skip over. Even if this is not strictly necessary, it might improve performance slightly since we skip a lot of operations. I will merge for now without removing this, we can always change it later!